src/Devices/PHControl.cpp

```diff
@@ -34,6 +35,13 @@ PHControl::PHControl() {
     targetPh = DEFAULT_PH;
     EEPROM_TC::instance()->setPH(targetPh);
   }
+  rampTimeEnd = EEPROM_TC::instance()->getRampTimeEnd();
+  if (rampTimeEnd == 0xFFFFFFFF || rampTimeEnd == 0) {
```

Comment:
If the end is set to a valid value, then we should load the other values. This is for the use case where the device is reset during the ramp. One also might consider clearing the values if we are beyond the end time.